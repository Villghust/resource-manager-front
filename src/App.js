import React from 'react';
import { Provider } from 'react-redux';

import MomentUtils from '@date-io/moment';
import { CssBaseline, ThemeProvider } from '@material-ui/core';
import { MuiPickersUtilsProvider } from '@material-ui/pickers';
import moment from 'moment';

import store from './providers/Store';
import Routes from './routes';
import 'moment/locale/pt-br';
import theme from './styles/muiTheme';

moment.locale('pt-br');

function App() {
    return (
        <MuiPickersUtilsProvider utils={MomentUtils} locale="pt-BR">
            <ThemeProvider theme={theme}>
                <Provider store={store}>
                    <div>
                        <CssBaseline />
                        <Routes />
                    </div>
                </Provider>
            </ThemeProvider>
        </MuiPickersUtilsProvider>
    );
}

export default App;
