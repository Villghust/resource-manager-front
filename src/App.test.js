import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

it('render correctly with no props', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
});
