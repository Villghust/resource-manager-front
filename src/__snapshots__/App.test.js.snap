// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly with no props 1`] = `
<div>
  <div
    className="makeStyles-root-3"
  >
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular makeStyles-toolbar-5 MuiToolbar-gutters"
      >
        <a
          className="makeStyles-root-6"
          href="/"
          onClick={[Function]}
        >
          <h6
            className="MuiTypography-root makeStyles-link-4 MuiTypography-h6"
          >
            Agenda
          </h6>
        </a>
        <a
          className="makeStyles-root-6"
          href="/users"
          onClick={[Function]}
        >
          <h6
            className="MuiTypography-root makeStyles-link-4 MuiTypography-h6"
          >
            Colaboradores
          </h6>
        </a>
        <a
          className="makeStyles-root-6"
          href="/resources"
          onClick={[Function]}
        >
          <h6
            className="MuiTypography-root makeStyles-link-4 MuiTypography-h6"
          >
            Recursos
          </h6>
        </a>
        <a
          className="makeStyles-root-6"
          href="/reports"
          onClick={[Function]}
        >
          <h6
            className="MuiTypography-root makeStyles-link-4 MuiTypography-h6"
          >
            Relat√≥rios
          </h6>
        </a>
      </div>
    </header>
  </div>
  <main
    className="makeStyles-content-2"
  >
    <div
      className="makeStyles-offset-1"
    />
    <div
      className="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <span
        className="MuiSkeleton-root MuiSkeleton-rect MuiSkeleton-pulse"
        style={
          Object {
            "height": 250,
            "width": undefined,
          }
        }
      />
    </div>
  </main>
</div>
`;
